utils/build-tooling-libs

```diff
@@ -245,6 +245,8 @@ class Builder(object):
             "-DSWIFT_BUILD_STATIC_STDLIB=FALSE",
             "-DSWIFT_BUILD_DYNAMIC_SDK_OVERLAY=FALSE",
             "-DSWIFT_BUILD_STATIC_SDK_OVERLAY=FALSE",
+            "-DSWIFT_BUILD_STDLIB_EXTRA_TOOLCHAIN_CONTENT=FALSE",
+            "-DSWIFT_BUILD_SYNTAXPARSERLIB=TRUE",
```

Comment:
Maybe group
```
"-DSWIFT_BUILD_SYNTAXPARSERLIB=TRUE",
```
along with 
```
cmake_args += [
            "-DSWIFT_BUILD_ONLY_SYNTAXPARSERLIB=TRUE",
            "-DSWIFT_BUILD_SYNTAXPARSERTEST=TRUE",
        ]
```
above, otherwise LGTM.

Reply:
Good idea. Changed it 👍